ts-client/src/vault/utils/index.ts

```diff
@@ -71,6 +83,33 @@ export const getOrCreateATAInstruction = async (
   }
 };
 
+export const getVaultPdas = async (
```

Comment:
should we separate all the utils into different file? e.g: `vault.ts`, `pool.ts`, `index.ts` (general)